wiki/Tournaments/Official_support/en.md

```diff
@@ -91,28 +91,27 @@ At the account support team's discretion, offending players may be issued timed
 
 ### Tournament screening
 
-A major part of being an officially supported tournament is access to the screening process. Similar to the "security checks" undertaken by all World Cup players, screening helps prevent people with recent serious infringements or tournament bans from disrupting play.
+A major part of being an officially supported tournament is access to the screening process. Similar to the "security checks" undertaken by all World Cup players, screening helps prevent users with recent serious infringements or tournament bans from disrupting play.
 
-Tournament organisers will be expected to provide a comma-separated list (or spreadsheet) including player usernames and user IDs. If the tournament is team-based, this list must reflect the grouping of players in their teams of play, complete with any team name or other identifying marker.
+Tournament organisers will be expected to provide a comma-separated list (or spreadsheet) including usernames and user IDs. If the tournament is team-based, this list must reflect the grouping of users in their teams of play, complete with any team name or other identifying marker.
 
 The comma-separated list should look like this:
 
 ```csv
-Player1,1234567
-Player2,1234567
-Player3,1234567
+User1,1234567
+User2,1234567
+User3,1234567
 ```
 
 For team-based tournaments, the expected format is:
 
 ```csv
-Player1,Team1,1234567
-Player2,Team1,1234567
-Player3,Team2,1234567
-Player4,Team2,1234567
+User1,Team1,1234567
+User2,Team2,1234567
+User3,Team3,1234567
```

Comment:
i mean this one

Reply:
Go with Leo's Format but replace Player with User.